### PR TITLE
V5 training data support

### PR DESCRIPTION
**Pull request by Tilps**
Except for the FRC variant.

### CODE REVIEW

File: tf/chunkparser.py [hunk 1]
```diff
@@ -204,15 +192,23 @@ def convert_v4_to_tuple(self, content):
...
             float32 root_q (4 bytes)
             float32 best_q (4 bytes)
             float32 root_d (4 bytes)
             float32 best_d (4 bytes)
+            float32 root_m (4 bytes)
+            float32 best_m (4 bytes)
+            float32 plies_left (4 bytes)
         """
-        (ver, probs, planes, us_ooo, us_oo, them_ooo, them_oo, stm, rule50_count, ply_count, winner, root_q, best_q, root_d, best_d) = self.v4_struct.unpack(content)
+        (ver, input_format, probs, planes, us_ooo, us_oo, them_ooo, them_oo, stm, rule50_count, dep_ply_count, winner, root_q, best_q, root_d, best_d, root_m, best_m, ply_count) = self.v5_struct.unpack(content)
```
Comment (Ttl) [inline tf/chunkparser.py]: `ply_count` could be renamed to `plies_left` as that's what it's referred to as elsewhere.

Comment (Tilps) [reply tf/chunkparser.py]: plies_left already used for the output variable, so I would want to rename that one if I rename ply_count to plies_left.  But I'm not sure what to rename that...

Comment (Ttl) [reply tf/chunkparser.py]: All other outputs are also named the same as variables unpacked from struct. I don't see a reason why it couldn't be `plies_left` everywhere.

Comment (Tilps) [reply tf/chunkparser.py]: Yeah, okay, seeems inline with the rest of the function so, done.

**Pull request merged**